docker-for-windows/troubleshoot.md

```diff
@@ -267,6 +267,15 @@ script](https://github.com/moby/moby/issues/24388).
 
 ### Virtualization
 
+In order for Docker for Windows to function properly your machine needs:
+
+1. [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/hyper-v-technology-overview) installed and working
+
+2. Virtualization enabled
+
+Note: If your OS is lower than Windows 10 PRO, you can use [docker-toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/)
```

Comment:
I don't think we should include this note here. If I were a new user I would think that this is a way for me to install Docker for Windows if I have an version lower than 10 Pro. But that is not the case. Toolbox is a completely different tech stack than Docker for Windows.

Also, [the Docker for Windows install page](https://docs.docker.com/docker-for-windows/install/#what-to-know-before-you-install) already explains that if you don't have the minimum requirements, you can use Docker Toolbox instead, and I think that's the right place for this info.

Reply:
That's a good point, that info can live elsewhere. Removed